starter/pom.xml

```diff
@@ -115,5 +115,11 @@
       <artifactId>jenvtest-fabric8-client-support</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Why this added?

Reply:
I had some errors when building the project locally, as the import of [jetbrains annotations](https://github.com/operator-framework/josdk-spring-boot-starter/blob/9d69dd0ce12e60fe8efea87d72b56e8db02a34a6/starter/src/main/java/io/javaoperatorsdk/operator/springboot/starter/CRDApplier.java#L10) was not working without this dependency.

Reply:
could you try to remove it and see if it fails on CI here?

Reply:
https://github.com/ebma16/josdk-spring-boot-starter/blob/cbbf449022cdeb2fba8e39da45af2a478e77bf88/starter/src/main/java/io/javaoperatorsdk/operator/springboot/starter/CRDApplier.java#L30

Can you remove the annotation pls from here?

Reply:
annotation removed -> CI is successful again